### DISCUSSION TITLE
Renovatebot overview doesn't expect a merge request for a package

### DISCUSSION THREAD

**Discussion by sagehassell** [renovatebot/renovate#10800] [Request Help] [answered]
I am trying to get renovatebot to update my ui-react library. I have set up a few custom configs to the initial renovate.json5 file to try and create a MR for ui-react, but when renovatebot updates the "What to Expect" section, it doesn't anticipate a MR for the ui-react library.

Here is my renovate.json5 file

{
      "$schema": "https://docs.renovatebot.com/renovate-schema.json",
      "extends": [
        "config:base",
        ":automergeMinor"
      ],
      "assignees": ["@shassell"],
      "packageRules": [
        {
          "matchPackageNames": [ "@qualtrics/ui-react", "@qualtrics/base-styles"],
          "groupName": ["UI-React and Styles"],
          "enabled": true
            },
        {
          "matchPackagePrefixes": ["@testing-library", "react"],
          "enabled": "false"
          }
      ]
  }

And this is what it currently says in my "what to Expect" section

![Screen Shot 2021-07-12 at 11 02 01 AM](https://user-images.githubusercontent.com/60795541/125327674-9852cb00-e300-11eb-9a88-cad52e580f49.png)

**Comment by viceice** [comment, marked as answer]
Check the debug logs, maybe some does are not public?

**Reply by sagehassell** [reply]
Okay, this is going to sounds silly, but where are the debug logs haha?

**Reply by viceice** [reply]
If you are using the hosted GitHub app, https://app.renovatebot.com

**Reply by sagehassell** [reply]
Ah, I see. Thanks! Turns out it was only changing devDependancies and not dependancies. I needed to change the rangeStrategy to "update-lockfile" so it will still update if the latest version matches the range in my package.json. Thanks for the help!
